MatterControlLib/PartPreviewWindow/View3D/View3DWidget.cs

```diff
@@ -1315,16 +1315,48 @@ public override void OnMouseUp(MouseEventArgs mouseEvent)
 				}
 				else // Allow right click on bed in all modes
 				{
-					// Workspace/plate context menu
-					UiThread.RunOnIdle(() =>
-					{
-						var popupMenu = new PopupMenu(ApplicationController.Instance.MenuTheme);
+					PopupBedActionsMenu(mouseEvent.Position);
```

Comment:
What's the reason for keeping this particular set of changes in? As far as I know we wanted to suppress cut/copy when right clicking with no hit target. For right click, it's more common to require a click directly on the object to expose the action. Try making a selection in windows explorer then right clicking off the selection